.gitignore

```diff
@@ -6,3 +6,9 @@ dist/
 maven.egg-info/
 
 *.pyc
+notes-maven*
```

Comment:
Cleaning this up, there shouldn't be personal stuff in .gitignore, this is a public package

Reply:
Also, I'm being careful about alphabetical/logical orderings here